css-three-column-layout/styles.css

```diff
@@ -0,0 +1,60 @@
+* {
+  color: white;
+  background-color: #1996da;
+  font-family: sans-serif;
+  font-size: small;
+}
```

Comment:
You generally want to avoid adding styles to the universal selector because now it's going to be applied to every single element, and if you want to change it, you need to override it. It would be better to apply these to the individual elements that need it